docs/articles/documentation/guides/advanced-guides/work-with-multiple-windows.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: docs
+title: Work with Multiple Windows
+permalink: /documentation/guides/advanced-guides/work-with-multiple-windows.html
+---
+
+# Work with Multiple Windows
```

Comment:
`Work with` is usually considered quite generic. `Test in Multiple Windows` can be better.